composer.json

```diff
@@ -10,6 +10,7 @@
         "blade-ui-kit/blade-icons": "^1.2",
         "blade-ui-kit/blade-ui-kit": "^0.3",
         "blade-ui-kit/blade-zondicons": "^1.3",
+        "codeat3/blade-simple-icons": "^1.39",
```

Comment:
I could not find any images of `discord` and `larachat` in neither the image folder of this project nor pre-installed blade-ui-kit package.
Therefore, I added one package `codeat3/blade-simple-icons` to get those images.